namespace/type-prefix-registry.md

```diff
@@ -0,0 +1,10 @@
+| Type Prefix | Class Name | Definition | Example | Reference |
+|-------------|------------|------------|---------|-----------|
+| SQ | Sequence | ... | `ga4gh:SQ:...` | [VRS Computed Identifers](https://vrs.ga4gh.org/en/1.2.0/impl-guide/computed_identifiers.html)|
```

Comment:
```suggestion
| SQ | Sequence | ... | `ga4gh:SQ.…` | [VRS Computed Identifers](https://vrs.ga4gh.org/en/1.2.0/impl-guide/computed_identifiers.html)|
```

(…and similarly throughout the table)

Reply:
Three periods would obviously be confusing next to a literal period character, but either way the ellipsis is just a placeholder until the examples (if they are retained) are filled in. But the comment is a reminder that the `:` should be a `.`.